sdk/monitor/monitor-opentelemetry-exporter/src/utils/spanUtils.ts

```diff
@@ -56,10 +57,22 @@ function createTagsFromSpan(span: ReadableSpan): Tags {
   }
   if (span.kind === SpanKind.SERVER) {
     const httpMethod = span.attributes[SemanticAttributes.HTTP_METHOD];
-    let httpClientIp = span.attributes[SemanticAttributes.HTTP_CLIENT_IP];
-    let netPeerIp = span.attributes[SemanticAttributes.NET_PEER_IP];
+    const httpClientIp = span.attributes[SemanticAttributes.HTTP_CLIENT_IP];
+    const netPeerIp = span.attributes[SemanticAttributes.NET_PEER_IP];
     if (httpMethod) {
-      tags[KnownContextTagKeys.AiOperationName] = `${httpMethod as string} ${span.name as string}`;
+      const httpRoute = span.attributes[SemanticAttributes.HTTP_ROUTE];
+      const httpUrl = span.attributes[SemanticAttributes.HTTP_URL];
+      tags[KnownContextTagKeys.AiOperationName] = span.name; // Default
+      if (httpRoute) {
+        tags[
+          KnownContextTagKeys.AiOperationName
+        ] = `${httpMethod as string} ${httpRoute as string}`;
+      } else if (httpUrl) {
+        try {
+          let url = new URL(String(httpUrl));
+          tags[KnownContextTagKeys.AiOperationName] = `${httpMethod} ${url.pathname}`;
```

Comment:
Will the path name change to `/` if empty?

Reply:
Yes